src/converter/internal/convertercontroller.cpp

```diff
@@ -181,7 +181,8 @@ mu::Ret ConverterController::convertPageByPage(INotationWriterPtr writer, INotat
     TRACEFUNC;
 
     for (size_t i = 0; i < notation->elements()->pages().size(); i++) {
-        const QString filePath = io::path_t(io::dirpath(out) + "/" + io::basename(out) + "-%1." + io::suffix(out)).toQString().arg(i + 1);
+        const io::path_t filePath_temp= io::path_t(io::dirpath(out) + "/" + io::completeBasename(out) + "-%1." + io::suffix(out));
+        const QString filePath = filePath_temp.toQString().arg(i + 1);
```

Comment:
I don't understand what you're struggling with. You can add just a newline at any point in the middle of a C++ statement. For example, just before the `=`.

```suggestion
        const QString filePath 
            = io::path_t(io::dirpath(out) + "/" + io::completeBasename(out) + "-%1." + io::suffix(out)).toQString().arg(i + 1);
```

Reply:
Ah, that's the point, I misunderstood the question, Thank you for answering me so patiently🥰

Reply:
But after I modified it according to your suggestion, it's still wrong🤨